server/app/utils/ConfigUtilities.java

```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import models.JiraAuth;
+import play.Configuration;
+import play.api.Play;
+
```

Comment:
Please add comment to explain what `Configuration` is this for the class and methods inside.